src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -847,12 +845,59 @@ public void shouldNotExtendLoanDueDateIfOverdueLoanIsRecalled() {
     ));
   }
 
+  @Test
+  public void shouldExtendLoanDueDateIfOverdueLoanIsRecalledAndConditionsAreMet() {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+
+    final Period alternateLoanPeriod = Period.weeks(3);
+
+    final Period loanPeriod = Period.weeks(1);
+    setFallbackPolicies(new LoanPolicyBuilder()
+      .withName("Can Circulate Rolling With Recalls")
+      .withDescription("Can circulate item With Recalls")
+      .rolling(loanPeriod)
+      .unlimitedRenewals()
+      .renewFromSystemDate()
+      .withAllowsAlternateRecallReturnInterval(true)
+      .withAlternateRecallReturnInterval(alternateLoanPeriod));
+
+    final DateTime loanCreateDate = now(UTC)
+      .minus(loanPeriod.timePeriod())
+      .minusMinutes(1);
+    DateTime expectedLoanDueDate = loanCreateDate
+      .plus(loanPeriod.timePeriod())
+      .plus(alternateLoanPeriod.timePeriod())
+      .plusMinutes(1);
+
+    final IndividualResource loan = checkOutFixture.checkOutByBarcode(
+      smallAngryPlanet, usersFixture.steve(), loanCreateDate);
+
+    requestsFixture.place(new RequestBuilder()
+      .recall()
+      .forItem(smallAngryPlanet)
+      .fulfilToHoldShelf()
+      .by(usersFixture.jessica())
+      .fulfilToHoldShelf()
+      .withPickupServicePointId(servicePointsFixture.cd1().getId()));
+
+    DateTime actualLoanDueDate = DateTime.parse(loansStorageClient.getById(loan.getId()).getJson().getString("dueDate"));
```

Comment:
Why is this using the due date from the loan in storage rather than the response from the check out API?

Reply:
We were following the paradigm layout out in the `shouldNotExtendLoanDueDateIfOverdueLoanIsRecalled` test.

Reply:
Hmm, ok, I don't know why that approach was used :-/ I'll resolve this as out of scope of this work.